src/select/__tests__/select.test.tsx

```diff
@@ -452,6 +452,17 @@ describe.each([false, true])('expandToViewport=%s', expandToViewport => {
     });
   });
 
+  describe('Inline Label', () => {
+    test('should render', () => {
+      const testLabel = 'Test label';
+      const { wrapper } = renderSelect({ inlineLabelText: testLabel });
+      const labelElement = wrapper.findByClassName(selectPartsStyles['inline-label']);
+      expect(labelElement).not.toBeNull();
+      expect(labelElement?.getElement()).toHaveTextContent(testLabel);
+      expect(labelElement?.getElement().tagName).toBe('LABEL');
+    });
+  });
```

Comment:
I think we should add another test which checks for the label -> trigger element association to ensuring its present.

```suggestion
    test('associate label with trigger button', () => {
      const testLabel = 'Test label';
      const { wrapper } = renderSelect({ inlineLabelText: testLabel });

      const labelForAttribute = wrapper
        .findByClassName(selectPartsStyles['inline-label'])!
        .getElement()!
        .getAttribute('for');
      const triggerId = wrapper.findTrigger().getElement()!.id;

      expect(labelForAttribute).toBe(triggerId);
    });
  });
```

Reply:
Agreed, thank you for providing the test code. Commited.